daphne/src/vdaf/mod.rs

```diff
@@ -40,7 +41,13 @@ pub(crate) enum VdafError {
     #[error("{0}")]
     Vdaf(#[from] prio::vdaf::VdafError),
     #[error("{0}")]
-    Uncategorized(String),
+    Dap(DapError),
+}
+
+impl VdafError {
+    fn fatal(e: impl ToString) -> Self {
+        Self::Dap(fatal_error!(err = ?e.to_string()))
+    }
 }
```

Comment:
This will hide the line number/module where this fatal error has been called. This macro should not be abstracted away. I think we need a better solution to this problem but I don't know one yet

Reply:
Oh, nice catch. And thanks for the info.

Reply:
Done.